ui/components/component-library/text/text.stories.tsx

```diff
@@ -286,29 +286,3 @@ export const Strong: StoryFn<typeof Text> = (args) => (
     </Text>
   </>
 );
-
-export const IsBrandEvolution = (args) => (
-  <>
-    {Object.values(TextVariant).map((variant) => {
-      // Wrap TextVariant.inherit in a span to show it's inherited styles
-      if (variant === TextVariant.inherit) {
-        return (
-          <Text as="span" {...args}>
-            <Text {...args} variant={variant} key={variant}>
-              {args.children || variant}
-            </Text>
-          </Text>
-        );
-      }
-      return (
-        <Text {...args} variant={variant} key={variant}>
-          {args.children || variant}
-        </Text>
-      );
-    })}
-  </>
-);
-
-IsBrandEvolution.args = {
-  isBrandEvolution: true,
-};
```

Comment:
We can remove this story documenting the removed `isBrandEvolution` feature flag prop